source/api/discovery/0.3/index.md

```diff
@@ -0,0 +1,784 @@
+
+#### 2.1.1. Level 0: Basic Resource List
+{: #level-0-basic-resource-list}
+
+The core information required to provide a minimally effective set of links to IIIF resources is just the URIs of those resources. However, with the addition of some additional JSON structure wrapped around those URIs, we can be on the path towards a robust set of information that clients can use to optimize their processing.  
+
+Starting with the IIIF resource URIs, we add an "Update" Activity wrapper around them.  The order of the resources in the resulting list is unimportant, but each should only appear once. In terms of optimization, this approach provides no additional benefit over any other simpler list format, but is compatible with the following levels which introduce significant benefits.  This is the minimum level for interoperability, while being compatible with the more detailed patterns described below.
+
+If resources are deleted after being referred to in the resource list, the entire list should be republished without the reference to the deleted resource. Clients should also expect to encounter resource URIs that are out of date and no longer resolve to a IIIF Manifest or Collection.
+
+Example Level 0 Activity:
+
+```
+{ 
+  "type": "Update",
+  "object": {
+    "id": "https://example.org/iiif/1/manifest",
+    "type": "Manifest"  	
+  }
+}
+```
+
+#### 2.1.2. Level 1: Basic Change List
+{: #level-1-basic-change-list}
+
+When dealing with large sets of resources, it can be useful to work with only those resources that have changed since the last time the list was processed. This can be facilitated by the addition of a time stamp that indicates when a resource was last modified or initially created. This is included using the `endTime` property, representing the time at which the activity of publishing the resource was finished. Lists with multiple activities are then ordered such that the most recent activities occur last. Consumers will then process the list of Activities in reverse order, from last to first, stopping when they encounter an Activity they have already processed in a previous run.
+
+Example Level 1 Activity:
+
+```
+{
+  "type": "Update",
+  "object": {
+    "id": "https://example.org/iiif/1/manifest",
+    "type": "Manifest"
+  },
+  "endTime": "2017-09-20T00:00:00Z"
+}
+```
+
+#### 2.1.3. Level 2: Complete Change List
+{: #level-2-complete-change-list}
+
+At the most detailed level, a log of all of the Activities that have taken place can be recorded, with the likelihood of multiple Activities per IIIF resource.  This allows the additional types of "Create" and "Delete", enabling a synchronization process to remove resources as well as add or update them. 
```

Comment:
"This allows" isn't clear. I suggest second sentence:

| \> Use of the additional types of "Create" and "Delete" allows explicit description of creations and deletions, enabling a synchronization process to remove resources as well as add or update them.

Reply:
:+1: